tcms/testruns/tests/test_views.py

```diff
@@ -334,6 +334,11 @@ def test_search_page_is_shown(self):
         response = self.client.get(self.search_runs_url)
         self.assertContains(response, '<input id="id_summary" type="text"')
 
+    def test_search_page_is_shown_with_get_parameter_used(self):
+        url = self.search_runs_url + '?product=1'
+        response = self.client.get(url)
+        self.assertContains(response, '<option value="1" selected>Kiwi</option>')
```

Comment:
Use ProductFactory to create a new product in setUpTestData and assert if it exists and is selected. 

There is not product named Kiwi b/c it has not been created and this is why test is failing.

Use full project name - Kiwi TCMS.

Reply:
The test is passing locally. The product is created in the grandparent class(BasePlanCase). I think at the first push the Scrutinizer was ok.

Reply:
Then use `self.product` and make sure the HTML you are asserting on is the same. Also try `html=True` parameter to the assert call.